src/imports/logger.js

```diff
@@ -1,6 +1,13 @@
 import winston from 'winston';
 
-// TODO: Configure logger
-const logger = winston;
+const logger = new winston.Logger;
+
+logger.add(winston.transports.Console, {
+  colorize: true,
+  level: 'debug',
```

Comment:
This can be a feature request, but we need to make this configurable with a flag. --verbose -v would turn this higher. --debug, etc... need to look at ember-cli or other tools to see what the best practices are here.